apis/v1beta1/microk8sconfig_types.go

```diff
@@ -33,6 +33,15 @@ import (
 // ClusterConfiguration contains cluster-wide configuration for a kubeadm cluster.
 type ClusterConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
+
+	// cluster agent port (25000) and dqlite port (19001) set to use calico port 179 and etcd port 2380 respectively
```

Comment:
is 179 a typo here?

Reply:
No this is not a typo. We move cluster agent to port 179 because this port is open on all nodes whereas the port 2379 was only open on the control plane nodes.

The 179 port is opened so that calico can be setup with bpf but in the MicroK8s case bpf is disabled. The 2379 port is used by etcd and so it was opened only on control plane nodes.